pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
I've just had a thought.  Would you like the ability to specify the content of file inline as an embedded API object?  It might make @ncdc's admission plugin more palatable.